src/main/scala/nl.knaw.dans.easy.deposit/DataFiles.scala

```diff
@@ -69,6 +69,7 @@ case class DataFiles(bag: DansBag) extends DebugEnhancedLogging {
         .withFilter(_._1.isChildOf(parentPath))
         .map((toFileInfo _).tupled)
         .toSeq
+        .sortBy(fileInfo => (fileInfo.dirpath, fileInfo.dirpath))
```

Comment:
you're sorting by `dirpath` twice???

Reply:
Somehow managed to push only half of the changes.